src/objects/SpriteText.js

```diff
@@ -0,0 +1,142 @@
+/**
+ * @author anhr / https://github.com/anhr/
+*/
+
+//A sprite based text component.
+//options:
+//{
+//	text: The text to be displayed on the sprite. Default is 'Sprite Text'
+//	position: THREE.Vector3 - position of the text. Default is new THREE.Vector3(0,0,0)
+//	textHeight: The height of the text. Default is 1
+//	fontFace: CSS font-family - specifies the font of the text. Default is 'Arial'
+//	fontColor: RGBA object or RGB object or HEX value. Default is 'rgba(255, 255, 255, 1)'
+//			Examples 'rgba(0, 0, 255, 0.5)', '#00FF00'
+//	bold: CSS font-weight. Equivalent of 700. Default is false.
+//	italic: CSS font-style. Default is false.
+//	fontProperties: string. Other font properties. The font property uses the same syntax as the CSS font property.
+//		Default is empty string. Example "900", "oblique lighter".
+//	center: THREE.Vector2 - The text's anchor point, and the point around which the text rotates.
+//		A value of (0.5, 0.5) corresponds to the midpoint of the text.
+//		A value of (0, 0) corresponds to the left lower corner of the text.
+//		A value of (0, 1) corresponds to the left upper corner of the text.
+//		Default is (0, 1).
+//	rect: text rectangle.
+//	{
+//		displayRect: true - the rectangle around the text is visible. Default is false
+//		backgroundColor: RGBA object or RGB object or HEX value. Default is 'rgba(100, 100, 100, 1)' - gray.
+//			Examples 'rgba(0, 0, 255, 0.5)', '#00FF00'
+//		borderColor: RGBA object or RGB object or HEX value. Default is 'rgba(0, 255, 0, 1)' - green
+//		borderThickness: Default is 5
+//		borderRadius: Default is 6
+//	}
+//}
+//Thanks to / https://github.com/vasturiano/three-spritetext
+function SpriteText( options ) {
```

Comment:
I don't think this file should be in the core. Please put it in your example.

Reply:
Now I have removed SpriteText.js because this file from my future pull request. I included this file in mistake. Sorry. See my commit https://github.com/mrdoob/three.js/pull/15434/commits/57876357c9efbf5de96a514312b49a91da0236aa